### PR TITLE
Updated examples/thoughts to 0.12.

### PR DESCRIPTION
**Pull request by Martinsos**

### CODE REVIEW

File: examples/thoughts/migrations/20240209145604_init/migration.sql
Comment (sodic) [inline examples/thoughts/migrations/20240209145604_init/migration.sql:1]: We didn't have migrations in the repo. Just making sure you know you're adding them in.

File: examples/thoughts/src/client/TopNavbar.jsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import React from "react";
+import { getUsername } from "wasp/auth";
```
Comment (sodic) [inline examples/thoughts/src/client/TopNavbar.jsx:1]: Should we reorder the imports?

**Review by sodic — APPROVED** [review]
Left two small comments

**Review by infomiho — APPROVED** [review]
Tested locally

**Comment by Martinsos** [comment]
> Tested locally

Hah tnx, although no need I also tested it locally of course!

File: examples/thoughts/migrations/20240209145604_init/migration.sql
Comment (Martinsos) [reply examples/thoughts/migrations/20240209145604_init/migration.sql:1]: I am looking at the code, and we had them already in the repo. I added a new one, along with existing migration(s), and this new one adds new Auth changes (Session), as you can see here.
I called the migration "init" by accident, my fingers were faster than my brain.

**Pull request merged**